source/_components/device_tracker.owntracks.markdown

```diff
@@ -13,11 +13,11 @@ featured: false
 ha_release: 0.7.4
 ---
 
-
 This platform allows you to detect presence using [Owntracks](http://owntracks.org/). OwnTracks allows users to track their location on iOS phones and publish it to an MQTT broker. This platform will connect to the broker and monitor for new locations.
 
 <p class='note'>
-The Android app for OwnTracks is no longer developed or supported, Zanzito is a drop in replacement for OwnTracks MQTT.
+The Android app for OwnTracks is no longer developed or supported, Zanzito is a drop in replacement for OwnTracks MQTT. 
```

Comment:
Development on the android app for owntracks has resumed.

Reply:
That is good to know! Maybe create a PR @MartinHjelmare? 🤔